src/components/wrapper/index.js

```diff
@@ -28,7 +28,7 @@ export const Wrapper = ({ loading, theme }) => (
     <div className="pipeline-wrapper">
       <LargePipelineWarning />
       <FlowChart />
-      <LoadingIcon className="pipeline-wrapper__loading" visible={loading} />
+      <LoadingIcon className="pipeline-wrapper__loading" visible={true} />
```

Comment:
I was wondering why this is turned on but just saw the description - no wonder. 
Don't forget to turn this off before merging 😆